tests/cucumber/features/step_definitions/shared_steps.coffee

```diff
@@ -0,0 +1,55 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
```

Comment:
Well, today it just worked, I must have been in a weird state with velocity yesterday.  Just pushed the fix.